test/action_controller/serialization_scope_name_test.rb

```diff
@@ -29,12 +29,12 @@ def render_new_user
 
   def test_default_scope_name
     get :render_new_user
-    assert_equal '{"users":{"admin?":false}}', @response.body
+    assert_equal '{"data":{"admin?":false,"id":"1","type":"test_users"}}', @response.body
```

Comment:
1. As noted above, it's `test_user` because that's the class of the resource, and I think it makes more sense to use that to determine the type, allowing one make generic serializers that decorate different models. But I am open to discussion on this. In this particular case, I think it would make sense to rename `TestUser` to `User` so that the type string is `users`.

2. I would prefer singular as well, but I decided to follow the examples in the spec which are all plural. But this is also why I wanted to make it very easy to change the inflection rules use.